lib/logger.js

```diff
@@ -0,0 +1,73 @@
+'use strict'
```

Comment:
curious - why can't we use our node logger here? I may have asked this before but I forget...

it really feels like we should be leaning on that to handle all of this rather than rewriting it. if there is something that prevents us from using it, what is it?

Reply:
I will after finishing testing